WebUI/Data/Seeder.cs

```diff
@@ -186,7 +186,7 @@ private static void SeedPreferences()
                 Genre = genre,
                 Platform = "Windows",
                 ReceivePromotionalEmails = false,
-                FavouriteGame = game,
+                //FavouriteGame = game,
```

Comment:
Why is this commented out?

Reply:
because it breaks otherwise (the cascade on delete stuff)

Reply:
Should it be deleted then?